### ISSUE TITLE
xorgxrdp 0.2.18 signature

### ISSUE THREAD

**Issue by bsmojver** [neutrinolabs/xorgxrdp#207]
Just making sure that release was signed with the old key on purpose. Paranoia etc.

Please let me know, so that I can build Fedora/EPEL packages.

**Comment by metalefty** [comment]
Hi, thanks for reporting. It was my mistake. I'll update the signature file shortly.

**Comment by metalefty** [comment]
@bsmojver It turned out the signature of v0.2.17 was wrong. I started to use my subkey `18AB838A907167745914871903993B4065E7193B` since v0.2.14. However, I signed v0.2.17 with my master key by mistake.  I believe v0.2.18 has been signed with the same (sub) key with v0.2.16, v0.2.15, v0.2.14. No worries, both keys are certainly mine.

- https://keyserver.ubuntu.com/pks/lookup?search=meta%40vmeta.jp&fingerprint=on&op=index

Sorry for my inexperience with GPG. Have your concerns been resolved?
